backend/main.py

```diff
@@ -0,0 +1,294 @@
+from flask import Flask, jsonify
+from flask_bootstrap import Bootstrap
+from flask_sqlalchemy import SQLAlchemy
+from flask_cors import CORS
+#from models import Item, Video, Reddit, Skill
+import json
+import requests
+import os
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = \
+        'postgresql://localhost/postgres'
+app.config['SQLALCHEMY_COMMIT_ON_TEARDOWN'] = True
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+CORS(app)
+db = SQLAlchemy(app)
+
+class Item(db.Model):
```

Comment:
What are your thoughts on splitting this out into a second file and calling it something like models? If you want me to do that make an issue and I'll do it later this week. No reason not to merge this because of it.